chat-plugins/scavengers.js

```diff
@@ -1,303 +1,629 @@
 /**
-* Scavenger hunts plugin. Only works in a room with the id 'scavengers'.
-* This game shows a hint. Once a player submits the correct answer, they move on to the next hint.
-* You finish upon correctly answering the third hint.
-* In an official hunt, the first three to finish within 60 seconds achieve blitz.
-*/
+ * Scavengers Plugin
+ * Pokemon Showdown - http://pokemonshowdown.com/
+ *
+ * This is a game plugin to host scavenger games specifically in the Scavengers room,
+ * where the players will race answer several hints.
+ *
+ * @license MIT license
+ */
 
 'use strict';
+
 const fs = require('fs');
-let scavengers = {
-	status: 'off',
-	blitz: null,
-	numblitzes: 0,
-	hints: null,
-	answers: null,
-	participants: {},
-	finished: [],
-	resultHTML: null,
-	updatinglb: false,
-};
 
-let scavengersRoom = Rooms.get('scavengers');
-if (scavengersRoom) {
-	if (scavengersRoom.plugin) {
-		scavengers = scavengersRoom.plugin;
-	} else {
-		scavengersRoom.plugin = scavengers;
+const Points = [20, 15, 10, 5, 1];
+const path = require('path');
+const DataFile = path.resolve(__dirname, '../config/chat-plugins/scavdata.json');
```

Comment:
By "capitalize", I mean that these constants would be named `POINTS` and `DATA_FILE`.

Like in https://github.com/Zarel/Pokemon-Showdown/blob/82ff7327fd3590d8903a586019df5a0fe30b2ab0/rooms.js#L14